src/pages/Schedules/Schedules.tsx

```diff
@@ -0,0 +1,40 @@
+import React, { useContext } from 'react'
+import Header from '../../components/Header/Header'
+import Button from '../../components/Button/Button'
+import Resume from '../../components/Resume/Resume'
+import SubTitle from '../../components/SubTitle/SubTitle'
+import AppContext from '../../Context/App.context'
+
+export default function Schedules() { 
+    const { setTime, setCurrentStep, time} = useContext(AppContext)
+    const buttons = [{label: "20:00", value: 20},
+                    {label: "21:00", value: 21},
+                    {label: "19:00", value: 19}, 
+                    {label: "18:00", value: 18}, 
+                    {label: "17:00", value: 17},]
+
+    const onClick = (e:Event) => {
+        e.preventDefault();
+        const value = (e.target as HTMLButtonElement).value;
+
+        if(setTime && setCurrentStep){
+            setTime(+value);
+            setCurrentStep(3);
+        }
+    }
+
+  return ( 
+    <div>
+        <Header/>
+        <Resume/>
+        <SubTitle sentence='Escolha a hora: '/>
+        <div className='time_buttons'>
```

Comment:
Also you add a class but not really have given any style to the buttons. does it looks like design?

you can give a className to the buttons, and add it as a prop and style it on schedules.scss
remember className naming convetions in this case would be: time-buttons__button.

or, you can pass a prop size to button, ex size='small' and style it in Button.scss

Also the container(time-buttons) needs a display flex respecting margin or paddings as per design.